README.md

```diff
@@ -71,3 +71,13 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## Differences between Home and Education sets
```

Comment:
Mention robot programs in the title?

Reply:
I can not think of a nice way to do this. Any ideas?

Reply:
Hmm... I don't have any either :grinning: What about putting it under a separate heading, perhaps "3rd-party index of demo robots"?

Reply:
It's nothing major; I'm happy with whatever you think is reasonable.

Reply:
The new section is just two sentences, and if we mention robots in the section header, the header becomes as informative as the whole section. So, unless @bittner has some other ideas, I would leave the header as is.